InternalPythonModules/android/cachelocation.py

```diff
@@ -142,9 +143,12 @@ def __findGeoLocationsInFile(self, file, abstractFile):
                     self._logger.log(Level.SEVERE, traceback.format_exc())
                     MessageNotifyUtil.Notify.error("Failed to index GPS trackpoint artifact for keyword search.", artifact.getDisplayName())
 
+        except SQLException as ex:
+            # Unable to execute Cached GPS locations SQL query against database.
+            pass
         except Exception as ex:
-            self._logger.log(Level.SEVERE, "Error parsing Cached GPS locations to blackboard", ex)
-            self._logger.log(Level.SEVERE, traceback.format_exc())
+            # Error parsing Cached GPS locations to blackboard.
```

Comment:
Why are we swallowing this? This could be something more critical that is not a sql exception.